docs/docs/concepts/streaming.md

```diff
@@ -9,6 +9,8 @@ There are several different modes you can specify when calling these methods (e.
 
 - [`"values"`](../how-tos/stream-values.ipynb): This streams the full value of the state after each step of the graph.
 - [`"updates"`](../how-tos/stream-updates.ipynb): This streams the updates to the state after each step of the graph. If multiple updates are made in the same step (e.g. multiple nodes are run) then those updates are streamed separately.
+- [`"custom"`](../how-tos/streaming-content.ipynb): This streams custom data from inside your graph nodes.
+- [`"messages"`](../how-tos/streaming-tokens.ipynb): This streams LLM tokens and metadata for the graph node where LLM is invoked.
```

Comment:
Somewhere here we need to explain that multiple streaming modes can be used at the same time -- without this something like "custom" is useless